frontend/packages/console-dynamic-plugin-sdk/src/index.ts

```diff
@@ -1,6 +1,11 @@
+import { VirtualizedGrid } from '../../console-shared/src/components/virtualized-grid';
```

Comment:
SDK should not import stuff from other packages directly. Only via `require` same as we do with `useK8sWatchResource` https://github.com/openshift/console/blob/master/frontend/packages/console-dynamic-plugin-sdk/src/api/api.ts#L9 and https://github.com/openshift/console/blob/master/frontend/packages/console-dynamic-plugin-sdk/src/runtime/plugin-api.ts#L5

There is also a PR which changes this a bit https://github.com/openshift/console/pull/9432 but no direct import from other packages will still apply